scheduler/cloudformation/cloudformation.go

```diff
@@ -192,6 +193,23 @@ func (s *Scheduler) SubmitWithOptions(ctx context.Context, app *scheduler.App, s
 	return tx.Commit()
 }
 
+func (s *Scheduler) Restart(ctx context.Context, app *scheduler.App, ss scheduler.StatusStream) error {
+	stackName, err := s.stackName(app.ID)
+	if err != nil {
+		return err
+	}
+	output := make(chan stackOperationOutput)
+	return s.updateStack(ctx, &updateStackInput{
```

Comment:
yea, i agree. i think for set/restart etc. we should wait, i'll do that in another PR